index.html

```diff
@@ -37,8 +45,32 @@ <h1 class="author">I’m Asif Amin Glad to see you!</h1>
     </section>  
      <!-- Work section -->
     <section class="work">
+      <!-- desktop-cards -->
+    <div class="desktop-card">
+      <div class="desktop-preview">
+        <img src="./assets/desktop/Snapshoot Portfolio.png" alt="Project 1" />
+      </div>
+      <div class="desktop-details">
+        <h2 class="desktop-title">Tonic</h2>
+        <ul class="desktop-info">
+          <li class="desktop-info_item">CANOPY</li>
+          <li class="desktop-info_item">Back End Dev</li>
+          <li class="desktop-info_item">2015</li>
+        </ul>
+        <p class="desktop-details_content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="desktop-catagories">
+          <li>Html</li>
+          <li>Css</li>
+          <li>Javascript</li>
+        </ul>
```

Comment:
- [ ]  I think it would be better also to make the technologies that were used to build the project clickable ``` buttons``` to match the Figma design 😂
- [ ] Kindly try to fix also the styling of these used technologies in the first card to be identical to others 😂 

**First card**
![image](https://user-images.githubusercontent.com/102757126/199966108-f5171ec5-74b4-41cd-842b-92ea20533d76.png)

**Third card**
![image](https://user-images.githubusercontent.com/102757126/199967731-9d69e80d-3fa7-45be-a492-623fcfe26eec.png)